### ISSUE TITLE
Not reporting

### ISSUE THREAD

**Issue by vinaybhavsar17** [hansemannn/titanium-crashlytics#27]
I've tried everything I could. Also followed the steps as it is mentioned in read me. But it is not reporting to crashlytics. The dashboard on firebase is still showing initial setup page, like it is not configured yet.
I am trying this on iOS.

![Screen Shot 2019-08-08 at 2 01 39 PM](https://user-images.githubusercontent.com/19553119/62687797-1ad00300-b9e5-11e9-9b3c-5f6186845ae5.png)

For crashing the app, I've written this code on click of a button,
Crashlytics.crash();
This doesn't crash the app immediately and it disconnects the app from Appcelerator studio and logging stops. But after few seconds, app is closed in the simulator. I am not sure if it crashed or something else happened.

Also there are lots of confusion in setting it up like,
1. What do we set in script-titanium-crashlytics.sh file if we are using firebase. Read me has only specified for fabric.
2. Will there be any difference in the content of script-titanium-crashlytics.sh file if we place our module in /Library/Application Support/Titanium/modules/iphone?
3. I am confused if the content in script-titanium-crashlytics.sh file would really start with ../../ or just ../
4. Read me says that define api key and secret key. But firebase gives us only api key and there is no secret key.

So basically I've tried every combination of settings but none of them worked and I am not sure how to proceed. Could you please help in this matter?

**Comment by pebosi** [comment]
Same problems with setup...

**Comment by guiu23** [comment]
Here you have info to find secret key: https://www.appypie.com/faqs/how-to-obtain-your-firebase-data-url-and-secret-key

**Comment by macasfaj** [comment]
I've followed this link to find secret key and it's not working :/ The App crashes on iOS.

**Comment by hansemannn** [comment]
Please the the latest version and ensure you test on a production environment as well!
